launch/simulation.launch.py

```diff
@@ -56,5 +52,80 @@ def generate_launch_description():
                 'verbose': 'true',
             }.items()
         )
+    
+    spawn_entity = Node(
+        package='gazebo_ros', 
+        executable='spawn_entity.py',
+        arguments=['-entity', my_neo_robot,'-file', robot_description_urdf,], 
+        output='screen')
+    
+    # if use_robot_state_pub is true, start the robot state publisher node
+    if use_robot_state_pub:
```

Comment:
why is there an `if` condition? robot_state_publisher should be always there..

Reply:
i will remove it anna...it was there in the older version of the code so I added a it as a launch argument...but now I'll completely remove it